### PR TITLE
[timeseries] Change verbosity of warnings for Chronos models

### PR DESCRIPTION
**Pull request by shchur**
*Issue #, if available:*

*Description of changes:*
- Clean up the logging & warning outputs for Chronos models at different verbosity levels

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: timeseries/src/autogluon/timeseries/models/chronos/model.py [hunk 1]
```diff
@@ -342,6 +348,7 @@ def _get_model_params(self) -> dict:
             logging_dir=str(output_dir),
             logging_strategy="steps",
             logging_steps=100,
+            disable_tqdm=True,
```
Comment (abdulfatir) [inline timeseries/src/autogluon/timeseries/models/chronos/model.py:351]: Why is tqdm always disabled?

File: timeseries/src/autogluon/timeseries/models/chronos/pipeline/utils.py [hunk 2]
```diff
@@ -317,7 +317,7 @@ def on_train_begin(self, args, state, control, **kwargs):
     def on_step_end(self, args, state, control, **kwargs):
         elapsed_time = time.monotonic() - self.start_time
         if elapsed_time > self.time_limit:
-            logger.info("\tStopping fine-tuning since time_limit is reached")
+            logger.log(15, "Stopping fine-tuning since time_limit is reached")
```
Comment (abdulfatir) [inline timeseries/src/autogluon/timeseries/models/chronos/pipeline/utils.py:320]: Why not `info`?

Comment (shchur) [reply timeseries/src/autogluon/timeseries/models/chronos/pipeline/utils.py:320]: Info (default) usually should only report the val_loss, fit time and predict time. Information about early stopping seems much more detailed, so we reserve it for verbosity=3

Comment (shchur) [reply timeseries/src/autogluon/timeseries/models/chronos/pipeline/utils.py:320]: Also, stopping training when the time_limit is reached is the default behavior for most models (e.g., all GluonTS models), and we don't usually log that.

File: timeseries/src/autogluon/timeseries/models/chronos/model.py [hunk 3]
```diff
@@ -15,6 +16,11 @@
 
 logger = logging.getLogger("autogluon.timeseries.models.chronos")
 
+# TODO: Replace `evaluation_strategy` with `eval_strategy` when upgrading to `transformers>=4.41` + remove warning filter
+warnings.filterwarnings("ignore", category=FutureWarning, message="`evaluation_strategy` is deprecated")
+# # TODO: Remove warning filter when upgrading to `transformers>=4.40`
```
Comment (abdulfatir) [inline timeseries/src/autogluon/timeseries/models/chronos/model.py:21]:
```suggestion
# TODO: Remove warning filter when upgrading to `transformers>=4.40`
```

**Review by abdulfatir — APPROVED** [review]
Left a small suggestion.

**Pull request merged**